tests/iz_importer_tests/image_fixture_creator.py

```diff
@@ -0,0 +1,108 @@
+import subprocess
```

Comment:
this is just a handy utility to generate the entry for the mock data json (and resize the image)